pkg/sfs/controllerserver.go

```diff
@@ -33,55 +37,74 @@ type controllerServer struct {
 	Driver *SfsDriver
 }
 
-func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
-	klog.V(2).Infof("CreateVolume called with request %v", *req)
-	if err := validateCreateVolumeRequest(req); err != nil {
-		return nil, status.Error(codes.InvalidArgument, err.Error())
-	}
-
+func (cs *controllerServer) CreateVolume(_ context.Context, req *csi.CreateVolumeRequest) (
+	*csi.CreateVolumeResponse, error) {
+	log.Infof("CreateVolume called with request %v", protosanitizer.StripSecrets(*req))
 	client, err := cs.Driver.cloud.SFSV2Client()
-    if err != nil {
-		klog.V(3).Infof("Failed to create SFS v2 client: %v", err)
-		return nil, status.Error(codes.Internal, err.Error())
-    }
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Failed to create SFS v2 client: %v", err)
+	}
+	name := req.GetName()
+	capacityRange := req.GetCapacityRange()
+	if err := createVolumeValidation(client, name, capacityRange); err != nil {
```

Comment:
if name and size are the same, it should return success

Reply:
done